.npmignore

```diff
@@ -11,3 +12,5 @@ circle.yml
 ARCHITECTURE.md
 CONTRIBUTING.md
 .editorconfig
+package-lock.json
+yarn.lock
```

Comment:
This should be checked in.

Reply:
Why? This is used by `npm` when publishing, not for `git`.

Lock files like `yarn.lock` do nothing in published packages, so shouldn't be included.

Reply:
Ah, you're right. Good catch!